velox/exec/Window.h

```diff
@@ -67,6 +77,8 @@ class Window : public Operator {
     const core::WindowNode::BoundType endType;
     const std::optional<column_index_t> startChannel;
```

Comment:
Are startChannel and endChannel needed anymore after your change for startFrameArg and endFrameArg ? If not, then changing their types from std::optional<column_index_t> to FrameArg should be sufficient.

Reply:
Apologies for missing this, I thought they were already removed.